app/models/rating.rb

```diff
@@ -0,0 +1,10 @@
+class Rating < ActiveRecord::Base
+  belongs_to :rater, :foreign_key => :rater_id, class_name: 'User'
+  belongs_to :ratee, :foreign_key => :ratee_id, class_name: 'User'
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.